### PR TITLE
Create 0000-01-02-Asanka98.md

### PR DESCRIPTION
**Pull request by Asanka98**
initial commit

### CODE REVIEW

**Review by AsankaW98 — APPROVED** [review]

**Comment by Asanka98** [comment]
done review

**Review by Asanka98 — COMMENTED** [review]
bcvbc

File: _posts/0000-01-02-Asanka98.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (Asanka98) [inline _posts/0000-01-02-Asanka98.md:5]: this is my line 5

**Review by Asanka98 — COMMENTED** [review]
done

**Pull request merged**